generators/app/__snapshots__/generator.spec.mts.snap

```diff
@@ -793,6 +794,7 @@ exports[`generator - app with gateway should match snapshot 1`] = `
   "gatlingTests": false,
   "generateBuiltInAuthorityEntity": true,
   "generateBuiltInUserEntity": true,
+  "generateLoginManagement": true,
```

Comment:
What is login management in this context?

Reply:
Authentication api. Any suggestion?

Reply:
`generateAuthenticationApi`?

Reply:
Yes, I think that makes more sense.